lib/screen/customer_support/support_thread_page.dart

```diff
@@ -680,7 +681,8 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
           MixpanelEvent.replyAnnouncement,
           data: {
             "id": payload.announcement!.announcementContextId,
-            "type": payload.announcement!.type
+            "type": payload.announcement!.type,
```

Comment:
it's too much force unwrapped null value here, can we assign a new var here: `announcement = payload.announcement!`